policy/modules/services/openvpn.te

```diff
@@ -1,4 +1,4 @@
-policy_module(openvpn, 1.18.0)
+policy_module(openvpn, 1.18.1)
```

Comment:
Don't increment the module version, this will be done when the PR is merged. (If multiple people create a pr for the same module it could end up causing merge conflicts)